.changeset/thirty-kings-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Update Hyperdrive logs to reflect that hyperdrive bindings are now supported in local dev
```

Comment:
please use the recommended [changeset message format](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changeset-message-format)

Reply:
@CarmenPopoviciu good to merge now?